src/java.base/share/classes/jdk/internal/util/random/RandomSupport.java

```diff
@@ -1663,15 +1653,15 @@ protected AbstractArbitrarilyJumpableGenerator() {
 
         // Methods required by class AbstractSpliteratorGenerator
 
-        public Spliterator.OfInt makeIntsSpliterator(long index, long fence, int origin, int bound) {
+        protected Spliterator.OfInt makeIntsSpliterator(long index, long fence, int origin, int bound) {
```

Comment:
Are those still needed? It looks like because of the missing `@Override` those could also be private?

Reply:
True that. There are also some mentions in the comments.

Reply:
Thank you.

Reply:
Please mark the CSR as reviewed.